app/src/hooks/useGetCurrentAvailableBalance.ts

```diff
@@ -3,9 +3,23 @@ import { useBalance as useBalanceL1 } from './useBalance';
 import { formatEther } from 'viem';
 import { useCurrentAccount } from '@iota/dapp-kit';
 import { useBridgeStore } from '../lib/stores';
-import { formatIOTAFromNanos } from '../lib/utils';
+import { formatIOTAFromNanos, parseAmount } from '../lib/utils';
+import { useBuildL1DepositTransaction } from './useBuildL1DepositTransaction';
+import { L1_BASE_GAS_BUDGET, L2_FROM_L1_GAS_BUDGET } from 'isc-client';
+import { useL2GasEstimate } from './useL2GasEstimate';
+import { MINIMUM_SEND_AMOUNT } from '../lib/constants';
+import { IOTA_DECIMALS } from '@iota/iota-sdk/utils';
 
-export function useGetCurrentAvailableBalance(): {
+const GENERIC_EVM_ADDRESS = '0x1111111111111111111111111111111111111111';
+const GENERIC_IOTA_ADDRESS = '0x1111111111111111111111111111111111111111111111111111111111111111';
```

Comment:
Is this really necessary? Can't you just play with the `enabled` setting of queries? like, `enabled: address.length > 0` ?